### DISCUSSION TITLE
Multi-sample analysis of treatment and control group

### DISCUSSION THREAD

**Discussion by synatkeamsk** [ImmuneDynamics/Spectre#177] [! News]
Dear Authors, 
First of all, I highly appreciate your team effector to create this beautiful work. I am wondering whether you could point me to the tutorial of how to analyze multiple samples of control and treatment group at the same times from transformation, gating, umap and generate proportion of subset of cell. 

in flowjo, we can gate on one and apply the gate to the rest of the group.. I am not sure whether how we can analyze multiple samples from different group at once using your package. I am just new to your package. 

Kind Regards,

Synat

**Comment by tomashhurst** [comment]
Hi @synatkeamsk , great question! By default, most of our workflows are intended to be used with multiple samples from varying groups. Check our the simple discovery workflow (https://immunedynamics.io/spectre/simple-discovery/) for an idea -- in this example, we have mock- and virally-infected samples of cells from mouse brains, and you can see how we do the comparisons. The fundamental idea is the data is all aggregated together at the start, clustered all together, and then we can start making comparisons after the fact. Does that make sense?
